src/acb/zeta.c

```diff
@@ -13,24 +13,6 @@
 #include "acb_poly.h"
 #include "acb_dirichlet.h"
 
-void
-acb_zeta_si(acb_t z, slong s, slong prec)
-{
-    if (s >= 0)
-    {
-        arb_zeta_ui(acb_realref(z), s, prec);
-    }
-    else
-    {
-        arb_bernoulli_ui(acb_realref(z), 1-s, prec);
-        arb_div_ui(acb_realref(z), acb_realref(z), 1-s, prec);
-        arb_neg(acb_realref(z), acb_realref(z));
-    }
-
-    arb_zero(acb_imagref(z));
-    return;
-}
-
```

Comment:
This was moved to `acb_dirichlet/zeta.c` since it was only used there, and is an internal function.